articles/build-a-ticketing-app-with-adonisjs-and-vuejs/index.md

```diff
@@ -0,0 +1,902 @@
+
+To view a single event, we created a route in the `routes.js` file and linked it with each event. The route is defined as below:
+
+```js
+.......
+import Ticket from "./views/Ticket.vue";
+
+......
+{
+    path: "/events/:id",
+    name: "event",
+    component: Ticket,
+  },
+......
+```
+
+The route is pointing to a `Ticket` view and we have already created that inside the `src/views` folders.
+
+```js
+    // src/views/Ticket.vue
+
+    <template>
+      <div class="text-center banner p-3 pb-5">
+        <div class="container pb-5">
+          <div class="row pb-5">
+            <div class="col-md-8 col-12 pb-5">
+              <div class="card card-custom p-5">
+                <h5 class="authBtn">{{ event.title }}</h5>
+                <hr />
+                <div class="container">
+                  <small class="authBtnInner pb-3">{{ event.description }}</small>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4 col-12">
+              <div class="card card-custom p-5">
+                <h5 class="authBtn">Event Ticket</h5>
+                <hr />
+                <span>Event Date:</span>
+                <small class="authBtnInner pb-3">
+                  {{ event.date }}
+                </small>
+                <hr />
+                <span>Event Price:</span>
+                <small class="authBtnInner pb-3">${{ event.ticket_price }}</small>
+                <router-link to="/" class="btn btn-primary customBtn"
+                  >Buy Ticket</router-link
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </template>
+    <script>
+    import { computed } from "vue";
+    import { useStore } from "vuex";
+    import { useRoute } from "vue-router";
+    export default {
+      name: "Ticket",
+      setup() {
+        const store = useStore();
+        const route = useRoute();
+        const event = computed(() => store.getters.getEvent(route.params.id));
+        return { event };
+      },
+    };
+    </script>
+```
+
+The Ticket component displays the events information including the Ticket price and a Buy button, which after purchasing an email will be sent to the purchaser including the ticket’s code.
+
+### Testing the project
+
+Here is a preview of what we have developed so far.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/qf7vvjNjjIU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This is a high-level abstraction of the entire codebase, you can clone the different repositories for this project and take a closer look at each of them in details.
+
+AdonisJS Backend Repository [here](https://github.com/Kaperskyguru/ticketing-system-adonisjs).
+<br />Vuejs 3 Frontend Repository [here](https://github.com/Kaperskyguru/ticketing-system-vuejs3).
+
+### Conclusion
```

Comment:
Add a `Further reading`/`References` section to reference other articles, which the reader could ponder upon